server/core/src/rodsServer.cpp

```diff
@@ -261,6 +263,10 @@ main( int argc, char **argv )
 
     init_logger(write_to_stdout, enable_test_mode);
 
+    irods::experimental::replica_access_table::init();
+    irods::at_scope_exit deinit_fd_table{[] { irods::experimental::replica_access_table::deinit(); }};
+    // TODO Consider looking at "serverExit" to deinit the replica access table.
```

Comment:
See comment

Reply:
this is likely ok because the init call performs cleanup before setting up the world.